webapp/components/LogInAccordion.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import { Typography, Button, Accordion } from '@material-ui/core';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Alert from '@material-ui/lab/Alert';
+
+
+
+import GetAppIcon from '@material-ui/icons/GetApp';
+
+
+import Input from './Input';
+
+import styles from './Form.module.css';
+import { fetchProfile, fetchToken } from 'service/medlemService';
```

Comment:
We should probably add some code formatting config/tool 😁

Reply:
Hehe, agree. Will create an issue, but attempt to clean up some stuff manually 😂